testdata/json-project/test-project-e2e.json

```
@@ -50,9 +50,75 @@
                         "super": "DocumentRepresentation",
                         "labels": {
                             "en": "PDF Resource"
+                        },
+                        "comments": {
+                            "en": "This resource has a pre-view permission."
                         }
                     }
                 ]
+            },
+            {
+                "name": "second-onto",
+                "label": "Second test ontology",
```

To make it more explicit:

```suggestion
                "name": "permissions-onto",
                "label": "Permissions ontology",
```